packages/theme-chalk/src/cascader.scss

```diff
@@ -116,49 +147,13 @@
     line-height: normal;
     text-align: left;
     box-sizing: border-box;
-
-    .#{$namespace}-tag {
-      display: inline-flex;
-      align-items: center;
-      max-width: 100%;
-      margin: 2px 0 2px 6px;
-      text-overflow: ellipsis;
-      background: getCssVar('cascader-tag-background');
-
-      &:not(.is-hit) {
-        border-color: transparent;
-      }
-
-      > span {
-        flex: 1;
-        overflow: hidden;
-        text-overflow: ellipsis;
-      }
-
-      .#{$namespace}-icon-close {
-        flex: none;
-        background-color: getCssVar('text-color', 'placeholder');
-        color: getCssVar('color-white');
-
-        &:hover {
-          background-color: getCssVar('text-color', 'secondary');
-        }
-      }
-    }
+    @include cascader-tag-normal(getCssVar('cascader-tag-background'));
   }
 
   @include e(collapse-tags) {
     white-space: normal;
     z-index: getCssVar('index-normal');
-    display: flex;
-    align-items: center;
-    flex-wrap: wrap;
-  }
-
-  @include e(collapse-tag) {
-    line-height: inherit;
-    height: inherit;
-    display: flex;
+    @include cascader-tag-normal(getCssVar('fill-color'));
```

Comment:
## Suggest

getCssVar('cascader-tag-background'), getCssVar('fill-color')

These two colors appear to be the same. How about unifying it into one?

<img width="236" alt="image" src="https://user-images.githubusercontent.com/27342882/202845864-da5bb285-1925-4d39-bab2-8bfdd5952881.png">

<img width="316" alt="image" src="https://user-images.githubusercontent.com/27342882/202845880-d890305d-972b-47f4-958f-d906ad9b65e7.png">

Reply:
Because the `--el-cascader-tag-background` variable can only be visible to elements beneath the `.el-cascader` elements, but now the tooltip is teleported to the `body`.

<img width="447" alt="image" src="https://user-images.githubusercontent.com/23100055/202846528-9072f511-b9e5-4db8-8bbd-af093502afac.png">

Reply:
Oh you are right. Thank you for the detailed explanation.